spec.bs

```diff
@@ -579,46 +580,75 @@ An auction config is a [=struct=] with the following items:
+  Opaque JSON data passed to the seller's script runner.
 : <dfn>seller timeout</dfn>
 :: A duration. Defaulting to 50.
+  Restricts the runtime of the seller's scoreAd() script.
 : <dfn>per buyer signals</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
   [=strings=].
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] are opaque JSON data
+  passed to corresponding buyer's script runner.
 : <dfn>per buyer timeouts</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
-  durations.
+  durations. 
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] restrict the runtime of
+  corresponding buyer's generateBid() script.
 : <dfn>all buyers timeout</dfn>
 :: A duration. Defaulting to 50.
+  Restricts generateBid() script's runtime of all buyers with unspecified timeouts.
 : <dfn>per buyer group limits</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
   non-negative 16-bit integers.
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] restrict the number of
+  bidding interest groups for a particular buyer that can participate in an auction.
 : <dfn>per buyer priority signals</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
   [=ordered maps=], whose [=map/keys=] are [=strings=] and whose [=map/values=] are {{double}}.
+  Per-buyer sparse vector that, along with a similar per-interest group sparse vector
+  [=interest group/priority vector=], has its dot product taken to calculate interest group
+  priorities. No signals key start with "browserSignals.", which is reserved for values coming from
+  the browser.
 : <dfn>all buyers priority signals</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=strings=] and whose [=map/values=] are
   {{double}}.
+  Merged with [=auction config/per buyer priority signals=] before calculating per-interest group
+  priorities. In the case both have entries with the same key, the entry in
+  `per_buyer_priority_signals` takes precedence. No signals key start with "browserSignals.", which
+  is reserved for values coming from the browser.
 : <dfn>all buyers group limit</dfn>
 :: A non-negative 16-bit integer.
+  Limit on the number of bidding interest groups for any buyer.
 : <dfn>component auctions</dfn>
 :: A [=list=] of [=auction config=]s.
+  Nested auctions whose results will also be fed to [=auction config/seller=]. Only the top level
+  [=auction config=] can have component auctions.
 : <dfn>seller experiment group id</dfn>
 :: A non-negative 16-bit integer.
+  Optional identifier for an experiment group to support coordinated experiments with the seller's
+  trusted server.
 : <dfn>all buyer experiment group id</dfn>
 :: A non-negative 16-bit integer.
+  Optional identifier for an experiment group.

```

Comment:
```suggestion
  Optional identifier for an experiment group to support coordinated experiments with buyers' trusted servers for buyers without a specified experiment group.
```

Reply:
Might be good to move "all buyer experiment group id" after "per buyer experiment group ids"